### PR TITLE
Embedded Login Updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** New section on verifying that the IC grant type is enabled and supporting notes throughout documentation.
- **Is this PR related to a Monolith release?** Yes 2022.10.0

Use the following link to review the main section that covers embedded login:

**Implement authorization by grant type guide - Interaction Code**
[Verify that the Interaction Code grant type is enabled](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/interactioncode/main/#verify-that-the-interaction-code-grant-type-is-enabled)

Use the following links to review the links to the above main section throughout the doc repo:

**Implement authorization by grant type - Interaction Code**

- [Set up your authorization server - second intro paragraph update](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/interactioncode/main/#set-up-your-authorization-server)

- [Enable Interaction Code grant on your authorization server (step 5 note)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/interactioncode/main/#enable-interaction-code-grant-on-your-authorization-server)

- [Set up your app (step 6 note)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/interactioncode/main/#set-up-your-app)

- [Enable the Interaction Code grant on an application (step 4 note)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/implement-grant-type/interactioncode/main/#enable-the-interaction-code-grant-on-an-application)

**Embedded authentication**:

- [Get set up - Set up your Okta org (step 2 link)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-org-setup/android/main/#get-set-up)

- [Get set up - Update the default authorization server (step 5 note)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-org-setup/android/main/#update-the-default-custom-authorization-server)

- [Get set up - Create a new application (step 5 note)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/oie-embedded-common-org-setup/android/main/#create-a-new-application)

**Auth JS fundamentals**:
[Create an app integration (steps 5&11 notes)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/auth-js/main/#create-an-app-integration)

**Embedded Okta Sign-In Widget**
[Create an app integration (steps 5&12 notes)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/embedded-siw/main/#create-an-app-integration)

**Sign in to your SPA embedded widget**
[Create an Okta app integration (steps 5&13)](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/guides/sign-in-to-spa-embedded-widget/angular/main/#create-an-okta-app-integration)

**Concept - Interaction Code grant type**
[Interaction Code flow - second intro paragraph update](https://6323ab85fcbd021d66da8251--reverent-murdock-829d24.netlify.app/docs/concepts/interaction-code/#the-interaction-code-flow)

### Resolves:

* [OKTA-520210](https://oktainc.atlassian.net/browse/OKTA-520210)

### CODE REVIEW

**Review by anweshdatla-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-theme-prose/global-components/VerifyICGrantType.vue [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <blockquote>
+    <p><strong>Note</strong>: If the <strong>Interaction Code</strong> checkbox doesn’t appear, the Interaction Code grant type isn’t enabled for your org. To enable it, go to <strong>Settings</strong> > <strong>Account</strong> > <strong>Embedded widget sign-in support</strong>. See <a href="/docs/guides/implement-grant-type/interactioncode/main/#verify-that-the-interaction-code-grant-type-is-enabled">Verify that the Interaction Code grant type is enabled</a>. <ApiLifecycle access="ea" /></p>
+  </blockquote>
+</template>
+
+<script>
+export default {
+  name: "VerifyICGrantType"
+};
+</script>
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-theme-prose/global-components/VerifyICGrantType.vue:11]: @susanharper-okta should there be a newline at end of file?

Comment (susanharper-okta) [reply packages/@okta/vuepress-theme-prose/global-components/VerifyICGrantType.vue:11]: doesn't need one, no - not for a vue file AFAIK.

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
+## Verify that the Interaction Code grant type is enabled
+
+<ApiLifecycle access="ea" />
+
+The Interaction Code grant type is a sign-in flow used by embedded applications to connect to Okta. The **Embedded widget sign-in support** setting controls this grant type for all OpenID Connect and API access management servers in your org. With this setting enabled, you can allow or deny apps the ability to use embedded sign-in flows across your entire org. See [Interaction Code grant type](/docs/concepts/interaction-code/) for detailed information on this grant type and how to use it.
+
+> **Note:** Super Admin permissions are required. If you don’t have those permissions for your org, contact your administrator.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md:7]: Should we link to the administrator roles docs in h.o.c.? I don't know how familiar our average developer-as-admin users are with the various roles.

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md:7]: https://help.okta.com/oie/en-us/Content/Topics/Security/administrators-super-admin.htm

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+
+1. To access this setting, from your Okta org's [Admin Console](/docs/concepts/okta-organizations/#admin-console), go to **Settings** > **Account** > **Embedded widget sign-in support**.
+1. Click **Edit**.
+1. If the checkbox isn’t selected, select it if you want to use the Interaction Code as a grant type for your OpenID Connect app integrations and authorization servers.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md:11]: I wonder if you need the conditional phrase, especially given the sentence after this one.
```suggestion
1. Select the checkbox if you want to use the Interaction Code as a grant type for your OpenID Connect app integrations and authorization servers.
```

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
 
-If your Okta org is enabled with Identity Engine, then the Interaction Code grant type is automatically configured in your [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server). However, if you’re using a [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server) with your app, you must enable the Interaction Code grant type for that authorization server.
+If your Okta org is enabled with Identity Engine, then the Interaction Code grant type is automatically configured in your [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server). For custom authorization servers that you are using with your app, verify that the Interaction Code grant type is an available option. If it isn’t, go to **Settings** > **Account** > **Embedded widget sign-in support**. See [Verify that the Interaction Code grant type is enabled](/docs/guides/implement-grant-type/interactioncode/main/#verify-that-the-interaction-code-grant-type-is-enabled) for more information on how to toggle the Interaction Grant type for your org. <ApiLifecycle access="ea" />
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md:30]: Just a suggestion.
```suggestion
If your Okta org uses Identity Engine, then the Interaction Code grant type is automatically configured in your [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server). For custom authorization servers that you are using with your app, verify that the Interaction Code grant type is an available option. If it isn’t, go to **Settings** > **Account** > **Embedded widget sign-in support**. See [Verify that the Interaction Code grant type is enabled](/docs/guides/implement-grant-type/interactioncode/main/#verify-that-the-interaction-code-grant-type-is-enabled) for more information on how to toggle the Interaction Grant type for your org. <ApiLifecycle access="ea" />
```

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/setup-app.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 4. Select either **Native Application**, **Single-Page Application**, or **Web Application** (depending on the type of application that you are working on) as the **Application type**, then click **Next**.
 5. Specify the **App integration name**.
 6. Select **Interaction Code** and **Refresh Token** as the **Grant type**.
+
+    > **Note:** If the **Interaction Code** checkbox doesn’t appear, the Interaction Code grant type isn’t enabled for your org. To enable it, go to **Settings** > **Account** > **Embedded widget sign-in support**. See [Verify that the Interaction Code grant type is enabled](#verify-that-the-interaction-code-grant-type-is-enabled) for more information on how to toggle the Interaction Grant type for your org. <ApiLifecycle access="ea" />
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/setup-app.md:5]: I fear I'm missing something. Wouldn't you use the global component here, too?

File: packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/setup-app.md [hunk 6]
```diff
@@ -15,5 +18,8 @@ If you have an existing app registered in Okta and want to enable the app to use
 1. In the Admin Console, go to **Applications** > **Applications**.
 2. Select your app from the application list.
 3. From the **General** tab, click **Edit** in the **General Settings** panel.
-4. In the **Grant type** > **Client acting on behalf of a user** option, select the **Interaction Code** check box (retain any other grant type that your app currently supports).
+4. In the **Grant type** > **Client acting on behalf of a user** option, select the **Interaction Code** checkbox (retain any other grant type that your app currently supports).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/setup-app.md:21]: I may be misunderstanding your parenthetical remark.
```suggestion
4. In the **Grant type** > **Client acting on behalf of a user** option, select the **Interaction Code** checkbox to retain any other grant type that your app currently supports.
```

**Review by brentschaus-okta — CHANGES_REQUESTED** [review]
Just a few questions and nits.

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

**Pull request merged**